docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: dependency-search
+append_help_link: true
+title: Dependency search 
+hide_title: true
+description: "Search through all your dependencies in all your onboarded repositories at any time."
+tags:
+  - Semgrep Supply Chain
+  - Team & Enterprise Tier
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+<ul id="tag__badge-list">
+{
+Object.entries(frontMatter).filter(
+    frontmatter => frontmatter[0] === 'tags')[0].pop().map(
+    (value) => <li class='tag__badge-item'>{value}</li> )
+}
+</ul>
+
+# Searching through your dependencies
+
+Semgrep Supply Chain's dependency search feature allows you to query for any dependency in your codebase at any time. This feature detects all transitive and direct dependencies across all of your repositories in Semgrep Cloud Platform (SCP). Dependency search lists all the versions of a dependency as well as the repositories that use the dependency.
```

Comment:
Optional suggestion:

```suggestion
Semgrep Supply Chain's dependency search feature allows you to query for any dependency in your codebase. This feature detects all transitive and direct dependencies across all of your repositories in Semgrep Cloud Platform (SCP). Dependency search lists all the versions of a dependency as well as the repositories that use the dependency.
```

Reply:
Or if any time means "for any version" - let's be a little bit more specific here... at any time may sound like: "Start your computer at any time you wish."

Reply:
The reason for "at any time" is because a vuln can be discovered without a rule or formal CVE and users need to find out right away if they use the dependency, as opposed to waiting for a rule to be available to detect it. This is spelled out in the 2nd paragraph.